### PR TITLE
[ADD] OCA Specific fields for Module maturity key

### PR DESCRIPTION
**Pull request by RoelAdriaans**
@elicoidal @sbidoul

### CODE REVIEW

File: github_connector_oca/views/odoo_module_version.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+Copyright (C) 2016-Today: Odoo Community Association (OCA)
+@author: Sylvain LE GAL (https://twitter.com/legalsylvain)
```
Comment (StephanRozendaal) [inline github_connector_oca/views/odoo_module_version.xml]: This is a new module, aren't you the author, @RoelAdriaans ?

**Comment by bizzappdev** [comment]
LGTM

**Review by StephanRozendaal — APPROVED** [review]

**Review by bizzappdev — APPROVED** [review]

File: github_connector_oca/models/odoo_module_version.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        selection=_ODOO_DEVELOPMENT_STATUS_SELECTION,
+        readonly=True,
+    )
+
+    @api.model
+    def manifest_2_odoo(self, info, repository_branch, module):
+        res = super(OdooModuleVersion, self).manifest_2_odoo(
+            info, repository_branch, module
+        )
+        res['development_status'] = info.get('development_status', False)
```
Comment (sbidoul) [inline github_connector_oca/models/odoo_module_version.py]: please apply `lower()` to the  `development_status` key.

**Review by elicoidal — APPROVED** [review]
LGTM

**Review by sbidoul — APPROVED** [review]
(code review only).

Do you plan to do something similar for the maintainers key?

**Comment by elicoidal** [comment]
> Do you plan to do something similar for the maintainers key?

Was not planed but could be discussed if it would increase the contributors' visibility

**Review by StephanRozendaal — APPROVED** [review]

**Comment by elicoidal** [comment]
@StephanRozendaal if you have time today after to include the maintainer key, be my guest :smile:

**Comment by StephanRozendaal** [comment]
@elicoidal  I'll take a look.

**Pull request merged**